### PR TITLE
Chore: consistent naming

### PR DESCRIPTION
**Pull request by nunopato**
- Prepends `hasura_graphql` to variables/flags related to Hasura.

### CODE REVIEW

**Comment by dbarrosop** [comment]
Problem with the change is that for the metadata we need the url `http://graphql-engine:8080/v1/metadata`, that's why I was using `v1` so either we introduce a new env var for the "metadata migration" or we will have to modify the URL in the code

**Comment by dbarrosop** [comment]
Re naming, I am good either way.

File: cmd/serve.go [hunk 1]
```diff
@@ -224,13 +224,13 @@ var serveCmd = &cobra.Command{
...
-			viper.GetString(graphqlEndpointFlag),
-			viper.GetString(hasuraMetadataAdminSecretFlag),
+			viper.GetString(hasuraEndpointFlag),
+			viper.GetString(hasuraAdminSecretFlag),
 			logger,
 		)
 
 		metadataStorage := getMetadataStorage(
-			viper.GetString(graphqlEndpointFlag) + "/graphql",
+			viper.GetString(hasuraEndpointFlag),
```
Comment (dbarrosop) [inline cmd/serve.go:233]: you need to add the `+ "/graphql"` you removed

Comment (nunopato) [reply cmd/serve.go:233]: true story, thank you.

**Pull request merged**